tkweb/settings/base.py

```diff
@@ -244,6 +244,9 @@
         'markdown.extensions.sane_lists',
         'markdown.extensions.admonition',
     ],
+    'extension_configs': {
+        'wiki.core.markdown.mdx.codehighlight': {'guess_lang': False},
```

Comment:
Skal være `'wiki.core.markdown.mdx.codehilite'` ('lite' i stedet for 'light')

Reply:
Tilføj evt. `'wiki.plugins.macros.mdx.toc': {'title': 'Indhold'}` hvis vi også ønsker en titel på TOC'en. (Ellers vil denne `'extension_configs'` dict overskrive standard-dicten som specificerer `toc` titel.)

Reply:
(Y)